src/mame/heathkit/tlb.cpp

```diff
@@ -93,6 +94,14 @@ DEFINE_DEVICE_TYPE(HEATH_ULTRA, heath_ultra_tlb_device, "heath_ultra_tlb", "Heat
 DEFINE_DEVICE_TYPE(HEATH_GP19, heath_gp19_tlb_device, "heath_gp19_tlb", "Heath Terminal Logic Board plus Northwest Digital Systems GP-19")
 
 
+
+device_heath_tlb_card_interface::device_heath_tlb_card_interface(const machine_config &mconfig, device_t &device) :
+	device_interface(device, "heathtlbdevice")
+{
+	m_slot = dynamic_cast<heath_tlb_connector *>(device.owner());
+}
```

Comment:
If you move setting `m_slot` to the initialiser list, you can make it a `const` pointer.